### PR TITLE
Strings: use documentation comments in objc-h template

### LINKED ISSUE #821: objc-h template should use documentation comments
I wanted to create PR with this change but contribution guide told me not to.

I think [this](https://github.com/SwiftGen/SwiftGen/blob/stable/templates/strings/objc-h.stencil#L40) line needs to be changed to generate documentation comment instead of regular comments. This way you can easily see key and base value for each property.

### PR DESCRIPTION
**Pull request by szotp**
Fixes #821.

### CODE REVIEW

**Comment by SwiftGen-Eve** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found some warnings in CHANGELOG.md</td>
    </tr>
  </tbody>
</table>

Hey 👋 I'm Eve, the friendly bot watching over SwiftGen 🤖

Thanks a lot for your contribution!



---

Once you fix those tiny nitpickings above, we should be good to go! 🙌

ℹ️ _I will update this comment as you add new commits_

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ _None_
   [David Jennes](https://github.com/djbe)
   [#562](https://github.com/SwiftGen/SwiftGen/issues/562)
   [#797](https://github.com/SwiftGen/SwiftGen/pull/797)
+* Strings: objc-h now emits documentation comments  
+  [@szotp](https://github.com/szotp)
```
Comment (SwiftGen-Eve) [inline CHANGELOG.md]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/72a38125677fd95e2a4fc395ab42994cb4df2a1b/CHANGELOG.md#L24"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>

Comment (szotp) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,9 @@ _None_
   [David Jennes](https://github.com/djbe)
   [#562](https://github.com/SwiftGen/SwiftGen/issues/562)
   [#797](https://github.com/SwiftGen/SwiftGen/pull/797)
+* Strings: objc-h now emits documentation comments
```
Comment (SwiftGen-Eve) [inline CHANGELOG.md]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/01ab71c3d006ee5eeed26f3c75050efbb7b6545a/CHANGELOG.md#L23"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>

**Review by djbe — APPROVED** [review]
Thx for the PR!

Note that Danger action is failing because this PR is from a non-SwiftGen repository. Will need to look into how to improve that.

**Pull request merged**